a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -21,22 +21,21 @@ test.serial('attempt to open vaults under the minimum amount', async t => {
 
   const mint = '3.0';
   const collateral = '5.0';
-  const error = await t.throwsAsync(() =>
-    openVault(USER1ADDR, mint, collateral),
-  );
-
-  t.true(
-    error?.message.includes(
-      'Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}',
-    ),
-    'Error message does not contain the expected text',
+  await t.throwsAsync(
+    () => openVault(USER1ADDR, mint, collateral),
+    {
+      message:
+        /Vault creation requires a minInitialDebt of {"brand":"\[Alleged: IST brand\]","value":"\[5000000n\]"}/,
+    },
+    'Error triggered by openVault does not contain the expected message',
```

Comment:
Your choice whether to leave it or not, but I don't think this adds much to the default error message which would also indicate that the resulting message didn't match.

Reply:
Changes addressed on commit [4216ee1](https://github.com/Agoric/agoric-sdk/pull/10123/commits/4216ee1d9c560d7581f600009a8a9de321aa43d0)